File: hastane-ekran-yonetimi/mobile/HospitalScreenManagement/models/Screen.kt
data class Screen(
    @SerializedName("_id") val id: String,
    @SerializedName("screenName") val screenName: String = "",
    @SerializedName("location") val location: String? = null,
    @SerializedName("isActive") val isActive: Boolean = true,
    @SerializedName("createdAt") val createdAt: String? = null,
    @SerializedName("__v") val version: Int = 0
) {
    override fun toString(): String {
        return "Screen(id='$id', screenName='$screenName', location=$location, isActive=$isActive)"
    }
} 

File: hastane-ekran-yonetimi/backend/models/Screen.js
// models/Screen.js
const mongoose = require('mongoose');

const screenSchema = new mongoose.Schema({
  screenName: { 
    type: String, 
    required: true,
    trim: true // Boşlukları temizler
  },
  location: { 
    type: String,
    trim: true
  },
  isActive: { 
    type: Boolean, 
    default: true 
  },
  currentPlaylist: {
    type: mongoose.Schema.Types.ObjectId,
    ref: 'Playlist'
  }
});

module.exports = mongoose.model('Screen', screenSchema);
